R/constants.R

```diff
@@ -149,3 +149,12 @@ emissions.TST_TO_TG <- 0.000907 # Conversion from thousand short tons to Tg
 emissions.NH3_HISTORICAL_YEARS <- 1990:2002
 emissions.NH3_EXTRA_YEARS <- 1971:1989
 emissions.EDGAR_YEARS <- 1971:2008
+
+# ======================================================================
+# Gas technologies
+
+energy.GAS_TECH <- c( "gas (steam/CT)", "gas (CC)" )
+
```

Comment:
Ok, I deleted the constant that was created and I pulled gas technologies and "whole sale" from the file "calibrated_techs". I created a tibble with this information, which is what I used to populate the  technology names and the minicam.energy.input.